bin/MadGraph5_aMCatNLO/cards/production/13TeV/DYJets_HT_LO_MLM/DYJets_HT_extralowmll/DYJets_HT-100to200/DYJets_HT-extralow_mll_100to200_proc_card.dat

```diff
@@ -0,0 +1,43 @@
+#************************************************************
+#*                        MadGraph 5                        *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 1.5.11                2013-06-21         *
+#*                                                          *
+#*    The MadGraph Development Team - Please visit us at    *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph 5                *
+#*                                                          *
+#*     run as ./bin/mg5  filename                           *
+#*                                                          *
+#************************************************************
+
+import model sm-ckm_no_b_mass
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
```

Comment:
Just to be sure... I copied the proc.dat card from:
   genproductions/bin/MadGraph5_aMCatNLO/cards/production/13TeV/DYJets_HT_LO_MLM/DYJets_HT_mll50/DYJets_HT-100to200/DYJets_HT-100to200_proc_card.dat

But it seems there is some duplication so I had a look at the manual and I'm going to change the process datacard to something like this (following this [suggestion](https://answers.launchpad.net/mg5amcnlo/+question/403756)):

    import model sm-ckm_no_b_mass
    define p = u c s d b u~ c~ s~ d~ b~ g
    define l+ = e+ mu+ ta+
    define l- = e- mu- ta-
    define j  = p
    # Specify process(es) to run
    generate p p > l+ l- / h @0
    add process p p > l+ l- j / h @1
    add process p p > l+ l- j j / h @2
    add process p p > l+ l- j j j / h @3
    add process p p > l+ l- j j j j / h @4
    # Output processes to MadEvent directory
    output DYJets_HT-extralow_mll_100to200 -nojpeg

Few more remarks

- I had to split the proposed HT bin because the gridpack production failed... now I propose to have HT 50-100, 100-200, 200-400, 400-600 and 600-Inf
- The HT bin 50-100 seems problematic, I'm trying now a minimum pt for leptons in the run card: 5  = ptl       ! minimum pt for the charged leptons
- The cross section I get from the finished grid pack are:
   - DYJets_HT-extralow_mll_200to400 Cross-section :   825.9 +- 1.486 pb 
   - DYJets_HT-extralow_mll_400to600 Cross-section :   68.77 +- 0.3156 pb
   - DYJets_HT-extralow_mll_600toInf Cross-section :   18.33 +- 0.4447 pb